chapters/equations.tex

```diff
@@ -506,6 +506,7 @@ \subsection{assert}\label{assert}
   \lstinline!condition! needs to be implicitly treated with \lstinline!noEvent! since otherwise events might be triggered that can lead to slightly changed simulation results.
   \end{nonnormative}
 \end{itemize}
+Tools may show more than the message for failed assertions, in particular the condition and the values of any variables used in it.
```

Comment:
I find this too weak for purposes of guiding how library authors should formulate assert messages.  Should they, or should they not explicitly include values of relevant variables in the message?  Should they even mention names of problematic model variables?

Reply:
I tried to extend that.

Reply:
Specifically I view it as two parts:
1. Models should just give a message. Nothing fancy - basically a literal string.
2. For users the information should be accessible, but I want to leave that open for tools to innovate.

Reply:
These are good objectives, but I think it needs to be at least normatively recommended:
```suggestion
Tools are recommended to provide more information than just the given message of a failed assertion, in particular the condition and the values of variables used in it.
```